apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClientImpl.cs

```diff
@@ -60,7 +66,11 @@ internal SubscriberClientImpl(SubscriptionName subscriptionName, IEnumerable<Sub
         settings.Validate();
         // These values are validated in Settings.Validate() above, so no need to re-validate here.
         _modifyDeadlineSeconds = (int)((settings.AckDeadline ?? DefaultAckDeadline).TotalSeconds);
+        // FIXME: This should be named autoExtendDelay. 
         var autoExtendInterval = TimeSpan.FromSeconds(_modifyDeadlineSeconds) - (settings.AckExtensionWindow ?? DefaultAckExtensionWindow);
+        // FIXME: This should be named autoExtendDelayForExactlyOnceDelivery.
+        // FIXME: MinimumAckExtensionWindowForExactlyOnceDelivery should be renamed to MinimumLeaseExtensionDelay or MinimumLeaseExtensionSchedule with the default value of 60 seconds.
```

Comment:
Unfortunately it's public - I think we'll need to introduce a new property, and mark the old one as obsolete (and explain in a comment). Fortunately it's read-only, so I don't think it will actually break anyone.